mail_inline_css/models/mail_template.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2017 David BEAL @ Akretion
+# Copyright 2019 Camptocamp SA
+
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+try:
+    from premailer import Premailer
+except (OSError, ImportError) as err:  # pragma: no cover
+    import logging
+
+    _logger = logging.getLogger(__name__)
+    _logger.debug(err)
```

Comment:
```suggestion
from premailer import Premailer
```

since it's already listed as external_dependencies, this try block seems unnecessary

Reply:
Hi, Thanks, i just updated it

Reply:
Thanks!